test/features/deploy_quarkus_service.feature

```diff
@@ -60,8 +60,8 @@ Feature: Deploy quarkus service
       | config | native      | <native> |
       | config | persistence | enabled  |
     And Kogito application "process-quarkus-example" has 1 pods running within <minutes> minutes
-    And HTTP GET request on service "process-quarkus-example" with path "orders" is successful within 3 minutes
-    And HTTP POST request on service "process-quarkus-example" with path "orders" and body:
+    And Service "process-quarkus-example" contains 0 instances of process with name "orders" within 3 minutes
+    And Start "orders" process on service "process-quarkus-example" with body:
```

Comment:
That could be replaced by one step:
`And Start "orders" process on service "process-quarkus-example" within 3 minutes with body:`

Reply:
Done